cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -340,6 +347,19 @@ func main() {
 	}
 
 	cfg.projectIdResource = fmt.Sprintf("projects/%v", *projectId)
+	if cfg.useRestrictedIps {
+		// manual.GenerateAndRegisterManualResolver generates a Resolver and a random scheme.
+		// It also registers the resolver. rb.InitialAddrs adds the addresses we are using
+		// to resolve GCP API calls to the resolver.
+		cfg.manualResolver, _ = manual.GenerateAndRegisterManualResolver()
```

Comment:
Why are we ignoring the error here?

Reply:
It's actually not an error, it's a cleanup function that isn't useful here.